### PR TITLE
Add DualSimTracer doc

### PR DESCRIPTION
**Pull request by Dolu1990**
Actualy used in NaxRiscv for day long sim

### CODE REVIEW

File: source/SpinalHDL/Simulation/bootstraps.rst [hunk 1]
```diff
@@ -189,3 +189,33 @@ For instance, to make the simulation fail after 1000 times the duration of a clo
...
+In the case you have a very long simulation, and you don't want to capture the wave on all of it (too bug, too slow), you have mostly 2 ways to do it.
+
+Either you know already at which ``simTime`` the simulation failed, in which case you can do the following in your testbench : 
+
+.. code-block:: scala
+    
+    disableSimWave()
+    delayed(timeFromWhichIWantToCapture)(enableSimWave())
+
+Either you can run a dual lock-step simulation, with one running a bit delayed from the the other one, and which will start recording the wave once the other simulation had a failure.
```
Comment (andreasWallner) [inline source/SpinalHDL/Simulation/bootstraps.rst:205]:
```suggestion
Or you can run a dual lock-step simulation, with one running a bit delayed from the the other one, and which will start recording the wave once the leading simulation had a failure.
```

File: source/SpinalHDL/Simulation/bootstraps.rst [hunk 2]
```diff
@@ -189,3 +189,33 @@ For instance, to make the simulation fail after 1000 times the duration of a clo
...
+
+To do this, you can use the DualSimTracer utility, which will ask you the compiled hardware, the window of time you want to capture the wave before falure, and a seed.
```
Comment (andreasWallner) [inline source/SpinalHDL/Simulation/bootstraps.rst:207]:
```suggestion
To do this, you can use the DualSimTracer utility, with parameters for the compiled hardware, the window of time you want to capture before failure, and a seed.
```

**Review by andreasWallner — CHANGES_REQUESTED** [review]
❤️ the idea!

**Comment by Dolu1990** [comment]
Thanks for the review :)

**Comment by mrcmry** [comment]
Deleted merged branch for cleanup.

**Pull request merged**